odl/discr/grid.py

```diff
@@ -689,80 +689,81 @@ def meshgrid(self):
         """
         return sparse_meshgrid(*self.coord_vectors)
 
-    def __getitem__(self, slc):
-        """Return ``self[slc]``.
+    def __getitem__(self, indices):
+        """Return ``self[indices]``.
 
         Parameters
         ----------
-        slc : `int` or slice
-            Negative indices and `None` (new axis) are not supported.
+        indices : `int` or slice
+            `None` (new axis) is not supported
```

Comment:
Perhaps dumb, but i think a word on "The index (dimension) that should be extracted" or something.